plugins/lowering/src/main/java/org/qbicc/plugin/lowering/VMHelpersSetupHook.java

```diff
@@ -17,13 +17,13 @@ public void accept(final CompilationContext ctxt) {
         RuntimeMethodFinder methodFinder = RuntimeMethodFinder.get(ctxt);
         // Helpers for dynamic type checking
         ctxt.enqueue(methodFinder.getMethod("arrayStoreCheck"));
-        ctxt.enqueue(methodFinder.getMethod("checkcast_class"));
-        ctxt.enqueue(methodFinder.getMethod("checkcast_typeId"));
-        ctxt.enqueue(methodFinder.getMethod("instanceof_class"));
-        ctxt.enqueue(methodFinder.getMethod("instanceof_typeId"));
-        ctxt.enqueue(methodFinder.getMethod("get_class"));
-        ctxt.enqueue(methodFinder.getMethod("classof_from_typeid"));
-        ctxt.enqueue(methodFinder.getMethod("get_superclass"));
+        ctxt.enqueue(methodFinder.getMethod("checkcastClass"));
+        ctxt.enqueue(methodFinder.getMethod("checkcastTypeId"));
+        ctxt.enqueue(methodFinder.getMethod("instanceofClass"));
+        ctxt.enqueue(methodFinder.getMethod("instanceofTypeId"));
+        ctxt.enqueue(methodFinder.getMethod("getClass"));
```

Comment:
This might return `Object.getClass`. Maybe we should just rename this one to `getClassForObject`?

Reply:
yep.  good catch. fixed